lib/cors_plug.ex

```diff
@@ -74,6 +74,10 @@ defmodule CORSPlug do
   # see: https://www.w3.org/TR/cors/#access-control-allow-origin-response-header
   defp origin(origins, conn) when is_list(origins) do
     req_origin = get_req_header(conn, "origin") |> List.first
-    if req_origin in origins, do: req_origin, else: "null"
+    if req_origin in origins do
```

Comment:
Hi @allonhadaya;
Thanks for the review and sorry for the late reply. I applied your recommendation. 

Cheers,
Mustafa